content/en/docs/Installation/trustedboot.md

```diff
@@ -55,10 +55,10 @@ To generate the Secure boot certificates and keys run the following commands:
 ```bash
 MY_ORG="Acme Corp"
 # Generate the keys
-docker run -v $PWD/keys:/work/keys -ti --rm enki genkey "$MY_ORG" -o /work/keys
+docker run -v $PWD/keys:/work/keys -ti --rm enki genkey "$MY_ORG" --expiration-in-days 365 -o /work/keys
```

Comment:
worth mentioning that this is the default value

Reply:
let's do that in another bunch of changes, got stuff in the queue that is stacking up quickly :)